test/tests/functional/pbs_basil_support.py

```diff
@@ -533,12 +533,10 @@ def test_pbs_alps_in_sync(self):
         try:
             rv = self.mom.log_match(
                 "This Cray system supports the BASIL 1.7 protocol.",
-                n='ALL')
+                n='ALL', max_attempts=10)
         except PtlLogMatchError:
-            self.logger.info(
-                "This Cray system not supports the BASIL 1.7 protocol.")
             self.skipTest(
-                reason='Test not applicable for systems not having BASIL 1.7')
+                reason='Test not applicable for  system not having BASIL 1.7')
```

Comment:
Is there two spaces between 'for' and 'system'?

Reply:
Done. Deleted extra space.